src/Main.hs

```diff
@@ -49,6 +49,6 @@ main = do
       bloom <- atomically $ newTVar bloom0
       eloop $ forConcurrently_
         (toList configBots)
-        (\(nick, bot) -> botThread bloom nick bot config)
+        (\(nick, bot) -> eloop $ botThread bloom nick bot config)
```

Comment:
Do we then still need the outer eloop?

Reply:
not sure, maybe stuff can break while parsing the json?

Reply:
Thats how it seems to me :laughing: